src/CompanyService.Business/Commands/Position/FindPositionsCommand.cs

```diff
@@ -1,29 +1,32 @@
-﻿using LT.DigitalOffice.CompanyService.Models.Dto;
-using System.Collections.Generic;
-using System.Linq;
+﻿using System.Linq;
 using LT.DigitalOffice.CompanyService.Data.Interfaces;
-using LT.DigitalOffice.CompanyService.Mappers.ResponsesMappers.Interfaces;
 using LT.DigitalOffice.CompanyService.Business.Commands.Position.Interfaces;
+using LT.DigitalOffice.CompanyService.Models.Dto.Responses;
+using LT.DigitalOffice.CompanyService.Mappers.Models.Interfaces;
 
 namespace LT.DigitalOffice.CompanyService.Business.Commands.Position
 {
     /// <inheritdoc cref="IFindPositionsCommand"/>
     public class FindPositionsCommand : IFindPositionsCommand
     {
         private readonly IPositionRepository _repository;
-        private readonly IPositionResponseMapper _mapper;
+        private readonly IPositionInfoMapper _mapper;
 
         public FindPositionsCommand(
             IPositionRepository repository,
-            IPositionResponseMapper mapper)
+            IPositionInfoMapper mapper)
         {
             _repository = repository;
             _mapper = mapper;
         }
 
-        public List<PositionResponse> Execute(bool includeDeactivated)
+        public FindPositionsResponse Execute(int skipCount, int takeCount, bool includeDeactivated)
```

Comment:
You can use FindResultResponse<> from kernel

Reply:
fixed